src/test/kotlin/com/exactpro/epfast/template/TemplateTests.kt

```diff
@@ -0,0 +1,889 @@
+package com.exactpro.epfast.template
+
+import com.exactpro.epfast.template.assertion.TemplatesComparison
+import com.exactpro.epfast.template.assertion.TemplatesComparison.assertTemplateListsAreEqual
+import com.exactpro.epfast.template.dsl.template
+import com.exactpro.epfast.template.xml.XmlTemplates
+import java.io.IOException
+import java.io.InputStream
+import org.junit.jupiter.api.Test
+
+class TemplateTests {
```

Comment:
Break it into several classes. I think it's better to divide by functionality we check: all elements, and different inheritance types (dictionary, templateNs, applicationNs).